widgetastic_manageiq/__init__.py

```diff
@@ -825,6 +825,115 @@ def workaround_save_issue(self):
             "{}._handlers.change.map(function(handler) {{ handler() }});".format(self.item_name))
 
 
+class SettingsGroupSubmenu(NavDropdown):
+    """
+    Widget for the Group 'Dropdown' menu that allows a user assigned to multiple groups
+    to change their active group dynamically Multiple group functionality added in 5.9
+    """
+    GROUP_SUBMENU = './/ul[contains(@class, "dropdown-menu scrollable-menu")]'
+
+    def __init__(self, parent, locator, logger=None):
+        Widget.__init__(self, parent, logger=logger)
+        self.locator = locator
+
+    def __locator__(self):
+        return self.locator
+
+    @property
+    def is_displayed(self):
+        return self.parent.expanded
+
+    @property
+    def expandable(self):
+        try:
+            self.browser.element('./.[contains(@class, "dropdown-submenu")]', parent=self)
+        except NoSuchElementException:
+            return False
+        else:
+            return True
+
+    @property
+    def expanded(self):
+        if not self.expandable:
+            return False
+        return self.browser.element(self.GROUP_SUBMENU, parent=self).is_displayed()
+
+    def expand(self):
+        """ Expand the Settings->Change Group submenu by hovering. """
+        if not self.expandable:
+            raise ValueError('{} is not expandable'.format(self.locator))
```

Comment:
Let's use a more specific exception here, I'd suggest:
https://github.com/RedHatQE/widgetastic.core/blob/master/src/widgetastic/exceptions.py#L17

Reply:
Will do. Most of the code for this class was copied from the parent class with a few changes to account for the fact that the group dropdown uses hover to expand instead of a click and xpath locator updates